tasks/main.yml

```diff
@@ -8,14 +8,14 @@
 # We are assuming all the projects use the same user.
 #
 # https://serverfault.com/questions/107187/ssh-agent-forwarding-and-sudo-to-another-user
-- name: Set up /etc/ssh/sshrc to open up the ssh auth sock to {{ project_user }} iffi the agent was forwarded
+- name: Set up /etc/ssh/sshrc to open up the ssh auth sock to {{ project_user }} iffi the agent was forwarded and the project user exists
   when: project_user is defined
   become: true
   lineinfile:
     path: /etc/ssh/sshrc
     backup: yes
     create: yes
-    line: if [ -n "$SSH_AUTH_SOCK" ] ; then setfacl -m u:{{ project_user }}:rx $(dirname "$SSH_AUTH_SOCK") -m u:{{ project_user }}:rwx "$SSH_AUTH_SOCK"; fi
+    line: if [ -n "$SSH_AUTH_SOCK" ] ; then if grep {{ project_user }} /etc/passwd ; then setfacl -m u:{{ project_user }}:rx $(dirname "$SSH_AUTH_SOCK") -m u:{{ project_user }}:rwx "$SSH_AUTH_SOCK"; fi; fi
```

Comment:
I'm not why, but on first runs with the ``ubuntu`` user, I would sometimes see errors like this:

> [WARNING]: sftp transfer mechanism failed on [xx.xx.xx.xx]. Use ANSIBLE_DEBUG=1 to see detailed information

But the deploy would still work, likely due to this [explanation](https://github.com/ansible/ansible/issues/22127#issuecomment-328178516).

However, the error goes away for me, so I think this is good to go 👍